src/io/stun.rs

```diff
@@ -683,10 +684,7 @@ impl<'a> Attributes<'a> {
 
         let username = self
             .username
-            .map(|v| {
-                let pad = 4 - (v.len() % 4) % 4;
```

Comment:
This should've been `(4 - (v.len() % 4)) % 4`. An alternative to this PR would be something smaller that just fixes this in the relevant places. 

Previously this would cause us to add 4 extra zero bytes for already aligned attributes.